content/summits/developer/2025/_index.md

```diff
@@ -128,13 +128,16 @@ scikit-HEP.
 - Kirstie Whitaker ([@KirstieJane](https://github.com/KirstieJane))
 - Kyle Sunden ([@ksunden](https://github.com/ksunden))
 - Madicken Munk ([@munkm](https://github.com/munkm))
+- Manfred Peter Fackeldey ([@pfackeldey](https://github.com/pfackeldey))
 - Matt Haberland ([@mdhaber](https://github.com/mdhaber))
 - Matthew Brett ([@matthew-brett ](https://github.com/matthew-brett))
 - Matthew Feickert ([@matthewfeickert](https://github.com/matthewfeickert))
 - Nabil Freij ([@nabobalis](https://github.com/nabobalis))
 - Ross Barnowski ([@rossbar](https://github.com/rossbar))
 - Sanket Verma ([@MSanKeys963](https://github.com/MSanKeys963))
```

Comment:
```suggestion
- Sanket Verma ([@sanketverma1704](https://github.com/sanketverma1704))
```
FYI: I've changed my GitHub username.

Reply:
Oh, yes. I was copy pasting it from previous years.

Reply:
sed-ed out all your old username in the new commit

Reply:
Thanks, @bsipocz!